docs/CONTRIBUTING.md

```diff
@@ -54,6 +54,10 @@ submitting a PR:
 - PR titles indirectly become part of the CHANGELOG so it's crucial to provide a good record of **what** change is being
   made in the title; **why** it was made will go in the PR description, along with a link to a GitHub issue if it
   exists.
+- <a id="breaking"></a>A breaking change is a change that forces users to change their code, command-line invocations,
```

Comment:
@silvanocerza 

> What should be considered a breaking change though?

I added a better description to to both the PR template and the docs borrowing words from @obra (Thanks :smile_cat: ). You like it?

@per1234 Nice suggestion about adding the anchor :+1: Please review the new wording like only you can do :detective:

Reply:
Yep, now it's clear what is meant.